requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask-APScheduler==1.12.2
 Flask-Caching==1.10.1
 Flask==2.0.1
+werkzeug==2.0.3
```

Comment:
as dependency for flask version 2.1.0 was coming which was breaking tests due to 'as_tuple' argument removal.